### ISSUE TITLE
Subcommand `metrics` should support `--metric=`  to configure what metrics to compute

### ISSUE THREAD

**Issue by weitsang** [nus-vv-streams/vvtk#14]
Even though we only support one metric for now, it is good to specify `--metric=gpsnr` in the example to be more explicit.

**Comment by YoungHS-NUS** [comment]
PCC Arena relies on [mpeg-pcc-dmetric](http://mpegx.int-evry.fr/software/MPEG/PCC/mpeg-pcc-dmetric) which has access control for users. My account is pending approval from your GitLab administrator and hence blocked.

**Comment by YoungHS-NUS** [comment]
Update:
I am still waiting for the approval from mpeg git lab admin

**Comment by YoungHS-NUS** [comment]
```bash
acd_rt,0.28488
acd_tr,0.14853
cd,0.21670
cd_psnr,66.34506
hd,11.11106
lc_psnr,46.94806
psnr_drms(fix resolution 1024),70.42217
vqoe,0.14426
```

these metrics are implemented
